index.html

```diff
@@ -31,7 +31,7 @@
 
 <body>
     <section>
-        <nav class="navbar navbar-expand-lg navbar-light" style="background: #2980b9;">
+        <nav class="navbar navbar-expand-lg navbar-light" style="background: linear-gradient(315deg, #045de9 0%, #09c6f9 74%);">
```

Comment:
Why have you modified the navbar?

Reply:
Undo the changes.

Reply:
I have just changed the color of the navbar.Should I undo that?

Reply:
Yes, no need for any other changes.

Reply:
Then you can merge my PR,right?

Reply:
Yes. Fix it.